pkg/devspace/configure/deployment.go

```diff
@@ -57,7 +58,7 @@ func GetDockerfileComponentDeployment(config *latest.Config, generatedConfig *ge
 			port = strconv.Itoa(ports[0])
 		} else if len(ports) > 1 {
 			port = survey.Question(&survey.QuestionOptions{
-				Question:     "Which port is the container listening on?",
+				Question:     "Which port is your application listening on?",
```

Comment:
This could lead to potential confusion since the function is also used for `devspace add deployment`. The user could think he now has to enter the port for the whole application instead the port for the single container he added is asked

Reply:
Not sure if application always perceived as the entire thing. I understand the concern more for `devspace add deployment` but honestly, no one uses that command and in `init` the question for container is confusing because there is no container, yet. At most, there is a Dockerfile. And I think asking for ports of containers that do not exist, may be confusing, especially if a user did not even have a Dockerfile beforehand.